server/views/licences/hdc_ap.pug

```diff
@@ -3,7 +3,7 @@ include includes/licenceHeader
 doctype html
 html(lang="en")
   head
-    link(href= "http://127.0.0.1:3000/public/stylesheets/forms-pdf.css?" + version, media="print", rel="stylesheet", type="text/css")
+    link(href= "http://127.0.0.1:3000/public/stylesheets/licences-pdf.css?" + version, media="print", rel="stylesheet", type="text/css")
```

Comment:
Ideally the IP address and port should not be hardcoded here... but appreciate this is legacy code. Maybe we could improve in the future.